documentation/development-processes-checklists.md

```diff
@@ -91,7 +103,7 @@ Upon completion of the testing phase, conduct the release procedure (outlined be
         - Include summary of `Merge hotfix/359 (#361) from master`.
         - Include the same description in the second line.
 
-### Release Procedures
+### Release Procedure
```

Comment:
they should all be singular, with the exception of `development procedures` simply because that one _is_ plural and references several procedures.